lib/graph/edge.js

```diff
@@ -701,6 +675,42 @@ Edge.prototype.coordAlongEdge = function (t, coords, display) {
   }
 }
 
+Edge.prototype.coordAlongOffsetEdge = function (t, coords, display) {
+  if (!this.baseRenderCoords) this.calculateBaseRenderCoords(display)
+
+  var len = 0
+  for (var i = 1; i < coords.length; i++) {
+    len += coords[i].len
+  }
+
+  var loc = t * len // the target distance along the Edge's base geometry
+  var cur = 0 // our current location along the edge (in world units)
+
+  for (i = 1; i < coords.length; i++) {
+    if (loc < cur + coords[i].len) {
+      var t2 = (loc - cur) / coords[i].len
+
+      if (coords[i].arc) { // arc segment
+        var r = coords[i].radius
+        var theta = Math.PI * coords[i].arc / 180
+        var ccw = Util.ccw(coords[0].x, coords[0].y, coords[1].x, coords[1].y,
```

Comment:
I may not fully understand, but isn't there a possibility that the arc is clockwise?